server/types/dps.ts

```diff
@@ -0,0 +1,53 @@
+import { PrisonerScheduleLenient } from './prisonApiImport'
```

Comment:
Why not put these in the `@types` folder instead of creating a new folder called `types`?

Reply:
Done